src/networkParams.ts

```diff
@@ -76,6 +76,23 @@ export class TransactionOptions {
         return new TransactionOptions(TRANSACTION_OPTIONS_TX_HASH_SIGN);
     }
 
+    /**
+     * Created a TransactionsOptions object with the setting for guarded transaction
+     */
+    static withTxGuardedOptions(): TransactionOptions {
+        return new TransactionOptions(TRANSACTION_OPTIONS_TX_GUARDED);
+    }
+
+    /**
+     * Returns true if Guarded Transaction Option is set
+     */
+    isGuardedTx(): boolean {
+        if ((this.value & TRANSACTION_OPTIONS_TX_GUARDED_MASK) == TRANSACTION_OPTIONS_TX_GUARDED) {
```

Comment:
Perhaps we can use a shorthand like `return (this.value & TRANSACTION_OPTIONS_TX_GUARDED_MASK) == TRANSACTION_OPTIONS_TX_GUARDED)`.